frontend/src/metabase/admin/upsells/components/Upsells.styled.tsx

```diff
@@ -55,9 +56,24 @@ export const UpsellCTALink = styled(ExternalLink)`
   }
 `;
 
-export const UpsellCardComponent = styled.div`
-  max-width: 200px;
-  box-sizing: content-box;
+interface Variants {
+  fullWidth?: boolean;
+  maxWidth?: number;
+}
+export const UpsellCardComponent = styled.div<Variants>`
+  ${({ fullWidth, maxWidth }) =>
+    fullWidth
+      ? css`
+          width: 100%;
+        `
+      : maxWidth
+      ? css`
+          max-width: ${maxWidth}px;
+        `
+      : css`
+          max-width: 200px;
+        `}
+  box-sizing: border-box;
```

Comment:
`box-sizing: content-box` with `width: 100%` causes horizontal scroll in embedding modal.

Here's when I use `box-sizing: border-box`
![image](https://github.com/metabase/metabase/assets/1937582/cc02e294-3ebf-4c22-96b5-dcaaf1251276)

Reply:
Fixed in https://github.com/metabase/metabase/pull/45424/commits/5d078c5d19dfbac003b5c6c612ab126169bd19f4

Thank you! I didn't think much, but it seems like we have some default CSS set already and `box-sizing: border-box` is one of them, so this is fine to be removed.